packages/gatsby-remark-images/src/gatsby-node.js

```diff
@@ -52,7 +54,43 @@ exports.pluginOptionsSchema = function ({ Joi }) {
       .description(
         `Additionally generate WebP versions alongside your chosen file format. They are added as a srcset with the appropriate mimetype and will be loaded in browsers that support the format. Pass true for default support, or an object of options to specifically override those for the WebP files. For example, pass { quality: 80 } to have the WebP images be at quality level 80.`
       ),
-    tracedSVG: Joi.boolean()
+    tracedSVG: Joi.alternatives()
+      .try(
+        Joi.boolean(),
+        Joi.object({
+          turnPolicy: Joi.string()
+            .valid(
+              // this plugin also allow to use key names and not exact values
+              `TURNPOLICY_BLACK`,
+              `TURNPOLICY_WHITE`,
+              `TURNPOLICY_LEFT`,
+              `TURNPOLICY_RIGHT`,
+              `TURNPOLICY_MINORITY`,
+              `TURNPOLICY_MAJORITY`,
+              // it also allow using actual policy values
+              Potrace.TURNPOLICY_BLACK,
+              Potrace.TURNPOLICY_WHITE,
+              Potrace.TURNPOLICY_LEFT,
+              Potrace.TURNPOLICY_RIGHT,
+              Potrace.TURNPOLICY_MINORITY,
+              Potrace.TURNPOLICY_MAJORITY
+            )
+            .default(Potrace.TURNPOLICY_MAJORITY),
+          turdSize: Joi.number().default(100),
+          alphaMax: Joi.number(),
+          optCurve: Joi.boolean().default(true),
+          optTolerance: Joi.number().default(0.4),
+          threshold: Joi.alternatives()
+            .try(
+              Joi.number().min(0).max(255),
+              Joi.number().valid(Potrace.THRESHOLD_AUTO)
+            )
+            .default(Potrace.THRESHOLD_AUTO),
+          blackOnWhite: Joi.boolean().default(true),
+          color: Joi.string().default(`lightgray`),
+          background: Joi.string().default(`transparent`),
+        })
+      )
```

Comment:
Also - maybe it's safer to remove defaults, because it's not this plugin that set them. My main motivation to add those defaults is that maybe they are (or will be) displayed in some form - if they are not, then better remove them as they can get out of sync